### PR TITLE
docs: Add configuring alt text using insertPattern

### LINKED ISSUE #42: Would be nice to have a feature to automatically set alt text to the file name
Hi love this extensions! Maybe I'm not using the settings correctly, but when I highlight a word then paste in the image it pastes in like this:

```
![](Repo_Layout.png)
```

but it would be really nice if the alt text could automatically be updated with the file name or path when pasting it in.

```
![Repo_Layout.png](Repo_Layout.png)
```

Thank you again for your hard work I love this extension makes documentation so much easier!

### PR DESCRIPTION
**Pull request by gojanpaolo**
Fixes #42

I also corrected some of the typos while at it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,9 +115,12 @@ Now you can enable `pasteImage.showFilePathConfirmInputBox` to modify file path
 
 - `pasteImage.insertPattern`
 
-    The pattern of string that would be pasted to text. 
-
-    You can use variable:
+    The pattern of string that would be pasted to text.
+    
+    You can configure both the alt text and the file path.
```
Comment (gojanpaolo) [inline README.md]: Here's the added documentation

File: README.md [hunk 2]
```diff
@@ -115,9 +115,12 @@ Now you can enable `pasteImage.showFilePathConfirmInputBox` to modify file path
...
+    For example: `![${imageFileNameWithoutExt}](${imageFilePath})`.
```
Comment (gojanpaolo) [inline README.md]: Here's the added documentation

Comment (mushanshitiancai) [reply README.md]: it would be better to add the explanation of `${imageFileName}` and `${imageFileNameWithoutExt}` to variables  list

Comment (gojanpaolo) [reply README.md]: I added a description of the example, if that's what you mean. :) The variable list below that line also describes those two variables.

Comment (mushanshitiancai) [reply README.md]: Oh, I've added it, I forgot.

**Comment by mushanshitiancai** [comment]
Thank you very much! Really appreciate all your efforts and your comments!

**Pull request merged**